src/NBthDEOptPara.cpp

```diff
@@ -12,172 +27,140 @@ using namespace roptim;
 // (useful for testing and development). The R code will be automatically
 // run after the compilation.
 //
+// original dnbinom_mu vector function - makes N separate calls to the built in
```

Comment:
Is this literally for storage of the old function used? If so, I would consider just removing. Let me know your thoughts?